source/SAMRAI/geom/CartesianCellDoubleConservativeLinearRefine.C

```diff
@@ -13,10 +13,19 @@
 #include <math.h>
 #include "SAMRAI/geom/CartesianPatchGeometry.h"
 #include "SAMRAI/hier/Index.h"
+#include "SAMRAI/pdat/ForAll.h"
 #include "SAMRAI/pdat/CellData.h"
 #include "SAMRAI/pdat/CellVariable.h"
 #include "SAMRAI/tbox/Utilities.h"
 
+#include "SAMRAI/tbox/AllocatorDatabase.h"
+
```

Comment:
We have static functions in tbox::MathUtilities to handle these kinds of things.  For example, tbox::MathUtilities<double>::Min(a,b);

If we need to use #define macros because we don't want to call static templated functions inside the for_all loops, please change them to SAMRAI_MIN, SAMRAI_ABS, etc., to avoid possible name conflicts with third-party libraries.  Also, MAX and SQRT are not used in this file, so can be deleted here.

Do we need a macro for copysign?  It's part of the standard.